mne/decoding/search_light.py

```diff
@@ -338,7 +358,10 @@ def _sl_score(estimators, X, y):
     """
     n_iter = X.shape[-1]
     for ii, est in enumerate(estimators):
-        _score = est.score(X[..., ii], y)
+        if scoring is not None:
+            _score = scoring(est, X[..., ii], y)
```

Comment:
@kingjr I made it with the signature `score_func(estimator, y, y_pred)` by using `make_scorer` so as to take care of the prediction type which `make_scorer` handles internally. However `cross_val_score` doesn't still pass. Let me know if you have any idea about it.